crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -123,8 +129,10 @@ impl Default for EmitJobRequest {
             invalid_transaction_ratio: 0,
             duration: Duration::from_secs(300),
             reuse_accounts: false,
-            transaction_type: TransactionType::P2P,
             mint_to_root: false,
+            transaction_mix: vec![(TransactionType::P2P, 1)],
```

Comment:
nit: It is more intuitive to make it a percentage rather than a fraction.

Reply:
you can put numbers that sum to 100, so it is percentage (i.e. 20, 30, 50). But you can also set (1,1) for even split, if you want. We use weights without forcing the sum in other places already, like leader election